src/_settings.scss

```diff
@@ -0,0 +1,763 @@
+ * **Mixins/Functions -> HiDPI background images**.
+ */
+
+$scally-g-hidpi-dpr: 1.3 !default;
+
+$scally-g-hidpi-dpi: ($scally-g-hidpi-dpr * 96) * 1dpi !default;
+
+$scally-g-hidpi-dppx: 1.3 * 1dppx !default;
+
+
+
+
+/* 7. HTML textual `input`s (Global)
+   ========================================================================= */
+
+$scally-g-textual-inputs: 'input[type="text"],
+                           input[type="search"],
+                           input[type="tel"],
+                           input[type="url"],
+                           input[type="email"],
+                           input[type="password"],
+                           input[type="month"],
+                           input[type="week"],
+                           input[type="time"],
+                           input[type="date"],
+                           input[type="datetime"],
+                           input[type="datetime-local"],
+                           input[type="number"]' !default;
+
+
+
+
+/* 8. Base (Local)
+   ========================================================================= */
+
+/**
+ * Booleans.
+ */
+
+$scally-base-apply-osx-font-smoothing: false !default;
+
+$scally-base-apply-link-transition: true !default;
+
+$scally-base-apply-style-to-disabled-and-readonly-states: true !default;
+
+$scally-base-apply-bottom-margin-to-paragraphs: false !default;
+
+$scally-base-apply-print-styles: true !default;
+
+
+/**
+ * Root element.
+ */
+
+$scally-base-root-element-background-color: $scally-g-color-white !default;
+
+
+/**
+ * Links.
+ */
+
+$scally-base-link-text-decoration: underline !default;
+
+$scally-base-link-text-decoration-on-hover: none !default;
+
+$scally-base-link-transition-duration: 0.15s !default;
+
+$scally-base-link-transition-timing-function: linear !default;
+
+
+/**
+ * Horizontal rule.
+ */
+
+$scally-base-hr-thickness: 1px !default;
+
+$scally-base-hr-style: solid !default;
+
+$scally-base-hr-color: $scally-g-color-text-base !default;
+
+
+/**
+ * Forms.
+ */
+
+$scally-base-disabled-and-readonly-state-opacity-strength: 0.6 !default;
+
+$scally-base-target-ios-selector: ".ios" !default;
+
+
+
+
+/* 9. Container (Local)
+   ========================================================================= */
+
+/**
+ * Booleans.
+ */
+
+$scally-container-apply-center-alignment: true !default;
+
+
+/**
+ * Widths.
+ */
+
+$scally-container-maximum-width: 1600 !default;
+
+$scally-container-minimum-width: 320 !default;
+
+// Wider
+$scally-container-maximum-width-large: ceil($scally-container-maximum-width * 1.5) !default;
+
+$scally-container-maximum-width-x-large: ceil($scally-container-maximum-width * 2) !default;
+
+// Narrower
+$scally-container-maximum-width-small: floor($scally-container-maximum-width / 1.5) !default;
+
+$scally-container-maximum-width-x-small: floor($scally-container-maximum-width / 2) !default;
+
+
+/**
+ * Gutter size.
+ */
+
+$scally-container-gutter: $scally-g-spacing-x-large !default;
```

Comment:
So in v2 the containers use a `%` width, e.g. `width: 94%` until they reach their max width. So if you're not setting the width to be `100%` you'll get vertical gutters on the left / right. For v3.0.0 the width is always `100%`, until the max width is reached, and for the vertical gutters we use a fixed size which by default is `$scally-g-spacing-x-large`. I feel using fixed gutters no matter the viewport width feels more in sync with the rest of the consistent'y applied vertical spacing.